main.py

```diff
@@ -1,43 +1,46 @@
-from os import error
+import sys, os, time, shutil, subprocess, json  # Base Libs
 
 
-while True: #If we get no import errors then break from the loop. If we do get an error install the dependencies and do the try/catch block again.
+while True:
      try:
-         import os,time,shutil,subprocess,json #Base Libs
          import spotipy
          from spotipy import util, SpotifyException
          from pynput.keyboard import Key, Controller
          break
      except ImportError:
          print("Import Error: Downloading off 'requirements.txt'")
          time.sleep(1)
-         os.system("pip3 install -r requirements.txt")
-         print("Done!")
+         # pip has zero return code only if successfull.
```

Comment:
I feel like we should remove such a feature altogether. Downloading requirements using pip from a file that has certain requirements is bad practice and should be avoided. Users should install the requirements before running the script. This can either be done by creating an installer in whatever language you prefer for the script (easiest would probably be a shell script) as well as by writing good documentation.

Reply:
True... I also thought about that. Never seen it in practice elsewhere.
I think it's better removed.